docs/guide/deploying.md

```diff
@@ -122,17 +124,52 @@ deploy:
     branch: main
 ```
 
-## GitLab Pages and GitLab CI
+### Actions
 
-1. Set the correct `base` in `docs/.vitepress/config.js`.
+1. Select a branch as GitHub pages source in repo settings. (By default you use the `gh-pages` branch below)
 
-   If you are deploying to `https://<USERNAME or GROUP>.gitlab.io/`, you can omit `base` as it defaults to `'/'`.
+2. Create a file named `deploy.yml` in `.github/workflow` of your project with the following content.
 
-   If you are deploying to `https://<USERNAME or GROUP>.gitlab.io/<REPO>/`, for example your repository is at `https://gitlab.com/<USERNAME>/<REPO>`, then set `base` to `'/<REPO>/'`.
+```yaml
+name: Deploy
+
+on:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set node version to 16.x
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+
+    - name: Install dependencies
+      run: npm install
+
+    - name: Build
+      run: npm run docs:build
+    
+    - name: Deploy
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: docs/.vitepress/dist
+```
+
+3. Now push commit to `master`, pages will deploy automaticlly
+
+## GitLab Pages and GitLab CI
 
-2. Set `outDir` in `.vitepress/config.js` to `../public`.
+1. Set `outDir` in `.vitepress/config.js` to `../public`.
 
-3. Create a file called `.gitlab-ci.yml` in the root of your project with the content below. This will build and deploy your site whenever you make changes to your content:
+2. Create a file called `.gitlab-ci.yml` in the root of your project with the content below. This will build and deploy your site whenever you make changes to your content:
```

Comment:
You can probably keep all the numbers as `1.` and let the Markdown parser handle it automatically:

```
1. Set `outDir`...
1. Create a file called...
```

Reply:
I think it's important to keep markdown well readable without a renderer. So I choose to number them manually.

Reply:
I don't think it makes much of a difference to readability, but I don't have a strong opinion here and will let the project's maintainers decide.